### PR TITLE
🏗 Macro Task Experiment in Canary and Production

### PR DESCRIPTION
**Pull request by kristoferbaxter**
I mistakenly did not add the `macro-after-long-task` experiment to the production configuration, and we're now awaiting this change to verify the impact of scheduler changes in production.

Fix for #24112.

### CODE REVIEW

File: build-system/global-configs/prod-config.json [hunk 1]
```diff
@@ -37,6 +38,7 @@
   "hidden-mutation-observer": 1,
   "ios-fixed-no-transfer": 0,
   "ios-scrollable-iframe": 0,
+  "macro-after-long-task": 1,
```
Comment (lannka) [inline build-system/global-configs/prod-config.json]: this will completely launch the feature in prod. I though you only want URL opt-in?

Comment (kristoferbaxter) [reply build-system/global-configs/prod-config.json]: You're absolutely right, fixed with 52fdceb.

File: build-system/global-configs/canary-config.json [hunk 2]
```diff
@@ -39,8 +39,8 @@
   "hidden-mutation-observer": 1,
   "ios-fixed-no-transfer": 1,
   "ios-scrollable-iframe": 0,
+  "macro-after-long-task": 1,
```
Comment (lannka) [inline build-system/global-configs/canary-config.json]: do you want to turn this on in canary? otherwise, please revert and we only need a prod cherrypick

Comment (lannka) [reply build-system/global-configs/canary-config.json]: oh noticed it's just a reorder

File: build-system/global-configs/prod-config.json [hunk 3]
```diff
@@ -37,6 +38,7 @@
   "hidden-mutation-observer": 1,
   "ios-fixed-no-transfer": 0,
   "ios-scrollable-iframe": 0,
+  "macro-after-long-task": 0,
```
Comment (lannka) [inline build-system/global-configs/prod-config.json]: you can leave this out to save a few bytes.

**Pull request merged**